Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift

```diff
@@ -529,6 +529,28 @@ extension SCGModels {
         var fallbackHandler: AddressInfoExtended?
         var version: String
     }
+
+
+    struct Chain: Decodable {
+        let chainId: String
```

Comment:
should it be Int?

Reply:
It's being sent as string from backend, so I thought to keep it string